kotlin/add-features-feature-service/README.md

```diff
@@ -1,33 +1,34 @@
-# Add features (Feature Service)
+# Add features
 
-Add new features to an online feature service.
+Add features to a feature layer.
 
-![Add Features Feature Service App](add-features-feature-service.png)
+![Image of adding features](add-features-feature-service.png)
 
 ## Use case
 
-A `FeatureLayer` created using this `ServiceFeatureTable` that is applied to the `ArcGISMap` will display any new Features automatically.
+An end-user performing a survey may want to add features to the map during the course of their work.
 
 ## How to use the sample
 
-Tap on the map to add a feature.
+Click on a location on the map to add a feature at that location.
```

Comment:
this is going to be one I know I'll be guilty of too: mentioning click vs tap! ;) 
```suggestion
Tap on a location on the map to add a feature at that location.
```